packages/core/src/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,59 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should accept component as a prop 1`] = `
+<DocumentFragment>
+  <section>
+    <header>
+      <div
+        data-testid="$heading.$props.$header"
```

Comment:
Couldn't find a better way to verify keys work properly and since this is crucial part of the library (I have some questions around it btw), I figured it might be better if we explicitly check for it.

Overall, not sure we're going to use testing library going forward, depends whether it's possible to check props/keys/whatever from user defined components as well